### PR TITLE
feature: import cluster with enabled workspace scheduler

### PR DESCRIPTION
**Pull request by iutx**
#### What type of this PR

Add one of the following kinds:
/kind feature

#### What this PR does / why we need it:
 import cluster with enabled workspace scheduler
testing:
image

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @luobily 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |   import cluster with enabled workspace scheduler           |
| 🇨🇳 中文    |   导入集群默认开启 workspace 调度           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/cmp/impl/clusters/import_cluster_test.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
...FiZ3BWWmtneU1XeERjVmhqWjNOa1dVdDVSVkZCUjJ4b2JISklOVUpEYVV4aVdGUnJaVUpZVlhWWmJtb3pVMG96ZEU5TlptZ3dTbWxOVkZGWWJXMXZVRUp6Q25GUFJ6RXdNVmhFZG05WFlVbGxWbFl5VUcwMVpsSjRhVGh2U3pJemVuTm5jM0pQVTJWcVkwTm5XVUkwZGxKSFYzUkVjVVZLUmxkSGQzWmhibGd5UlRZS01sVjNTVGhwYVROTU1scGlSemRrVUVaME5rbDJUMGxPWW5OMlNFWk5iVXRsTkZkcmEzSkdZMVJ0V0RoMU4zcFJhMDFNWjFWVVVTOVhSek5OVm5GWU5RcHRibGRIVm01VWVsVm1aRFUxWW5ZMWJsbEZjbTV3VGtob1VrcFZjelY0ZVc5RFFrMXJlVTlTUldGRFJUZzRhMll3Tms4NGVqaFNVMm81V1RaU01saDVDbVUwZEVWR1dIUm9ORlZtUlRZNVRVTkZSRVp4Tm5kTFFtZEhjRzl3VDFnellsWXpVMXBhVFV4MFlXZDVWWFZzT1VkUWRGVTBSWFY2Vnk5MGJ5OVNPR0lLVGxOSVFuVm9PVmhRT0dSSUwzRTRaMnRPVWxwcWEzUlZiakpuZVRKS1lWRkhPU3RMWkVkd1RUSlFWRTVCVDAwemVGWkRLMGxCV2xaeFEzVmFSV3hXVHdvM1MyNHdlRTVIVjNwdFkxcHpZa0ZETTJGUllubDBaV292SzNnd1RGbGFTRloyWjJoSFVtNWhjQzh5Ynk5dE1tZFJXVFZLVEZKa01WbGpOMDA1UmxOS0NrZHViblpCYjBkQlYzbFVWbGhFVlhGUVdXaFNVRlZ1VTBNdmRsUTRNV05MVW5GVmJHdEllRVYyTkVJeWVWcEZLM05oVVdJNFVEQlZRemRDTWt4TFUzUUtabmt6ZG5oVWRVMHhlRVpsV2t3emVHUllXRVZzWlVWTFVXSlBVM1JXY2tJMlVXVjRkRTVyY0dsM1MwTlVTVlJOUTJFM0wwazVlRzhyVVhBMFVqZHJiUXBJYmxwdlRFNVdkbEZtUkZoTFF6VnNNakpKYjNjclVIcHRaMmRtV1ZNNE1qUkhaVmRJUldKdWFtRkRiRE5pT0ZSb2FuYzlDaTB0TFMwdFJVNUVJRkpUUVNCUVVrbFdRVlJGSUV0RldTMHRMUzB0Q2c9PQo=",
```
Comment (sfwn) [inline modules/cmp/impl/clusters/import_cluster_test.go:64]: Does this base64-encoded data is sensitive or not?

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/cmp/impl/clusters/import_cluster_test.go (hunk 1 above)
Comment (iutx) [reply modules/cmp/impl/clusters/import_cluster_test.go:64]: The unit test data was all from the local temporary KIND environment.

**Review by sfwn — APPROVED** [review]

**Pull request merged**